src/app/Fake.netcore/Cli.fs

```diff
@@ -7,84 +7,100 @@ open Fake.Core.CommandLineParsing
 
 let fakeArgsHint =
   """
-General:
-
-  The Fake command line is divided into runtime and script arguments.
-  Runtime arguments control compilation and processing of the script,
-  while script arguments are specific for the script or provided by
-  a NuGet package.
-  In most use cases you use the "Fake.Core.Target"-Package and therefore
-  inherit the corresponding command line interface. While these arguments
-  are not strictly part of the runtime we still show both below to
-  make it easier for newcomers.
-
- -- RUNTIME ARGUMENTS SECTION --
+FAKE 5
```

Comment:
Thanks for the clean-up and organizing the help message!
I would like to suggest that we automate this and get the version of the current FAKE-CLI and print it instead of
hard-coding it as `FAKE 5`